### ISSUE TITLE
allow simulating fires only after some time during the simulation

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/scfm#28]
Would allow use of static fire perimeter maps to be fed to the simulation to allow using historic fire data for years where we have data instead of simulating those fires (e.g., via https://github.com/PredictiveEcology/historicFires).

- [ ] implement `.runInitialTime` following approach in `fireSense`;
- [ ] ensure all outputs and other objects are standarized.

related: https://github.com/CeresBarros/FavierFireSpread/issues/1#issuecomment-1295301928

**Comment by ianmseddy** [comment]
I think this would be easy to implement - just add one extra scheduling param to igntion/escape/spread with default start(sim). I'll do that next week

**Comment by achubaty** [comment]
yup, it's an easy add -- it exists in fireSense so we can follow similar approach / use same param names for consistency. thanks!

**Comment by ianmseddy** [comment]
So all three already have `startTime` as a param, which does exactly what you want. Do you want me to rename this `.runInitialTime`?

**Comment by achubaty** [comment]
no, that's great. love that these old issues were already resolved accidentally by us ;)
